src/thermopro_ble/parser.py

```diff
@@ -80,12 +80,17 @@ def _start_update(self, service_info: BluetoothServiceInfoBleak) -> None:
         if name.startswith(("TP96", "TP97")):
             # TP96 has a different format
             # It has an internal temp probe and an ambient temp probe
-            (
-                probe_zero_indexed,
-                internal_temp,
-                battery_voltage,
-                ambient_temp,
-            ) = UNPACK_SPIKE_TEMP(data)
+            try:
+                (
+                    probe_zero_indexed,
+                    internal_temp,
+                    battery_voltage,
+                    ambient_temp,
+                ) = UNPACK_SPIKE_TEMP(data)
+            except struct_error:
+                _LOGGER.error(f"Error parsing data from probe: {data}")
```

Comment:
```suggestion
                _LOGGER.error("Error parsing data from probe: %s", data)
```

Please follow https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-not-lazy.html

Reply:
Done